client/index.jsx

```diff
@@ -0,0 +1,18 @@
+import { h, render } from "preact";
+
+import Reporter from "./components/Reporter";
+
+import "./style/index.css";
+
+window.addEventListener("load", () => {
```

Comment:
I wanted the app to execute only during the `onload` function. Is there anything wrong in having this, since this is a standalone file.